staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -95,6 +96,8 @@ func (b *backend) processEvents(ev ...*auditinternal.Event) error {
 		list.Items = append(list.Items, *e)
 	}
 	return b.w.WithExponentialBackoff(func() rest.Result {
+		trace := utiltrace.New(fmt.Sprintf("Audit events webhook request: %s", b.name))
+		defer trace.LogIfLong(100 * time.Millisecond)
```

Comment:
remote audit logging happens after REST handlers finish, this threshold doesn't have to be bounded by the one for REST handlers

Reply:
Gotcha. 100ms seemed appropriate, mainly for consistency across webhooks, let me know if you feel a different limit would be better.